docs/changes/cosmology/12966.feature.rst

```diff
@@ -0,0 +1,2 @@
+The new module ``cosmology/tests/helper.py`` has been added to provide tools
+for testing the cosmology module and related extensions.
```

Comment:
Are updates in the testing infrastructure supposed to have change log entries?

Reply:
A good question. Generally no, but this is intended to be public API, as I'm trying to open up the tests more. So maybe?